src/modules/tests/editRemoveAll.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+import LocalStorage from '../localStorage.js';
+import { addTask } from './__mocks__/addRemove.js';
+import { toggleStatus, deleteAllCompleted } from './__mocks__/editRemoveAll.js';
+
```

Comment:
- [ ] Great work implementing the requirements, however, you didn't create a test for updating the task description, kindly add a test for updating the description